frontend/src/components/pages/listings/index/Listings.tsx

```diff
@@ -54,11 +56,24 @@ const Listings: React.FC<{
         </Grid>
       )}
       {data &&
+        data.listings.length > 0 &&
         data.listings.map((listing) => (
           <Grid container item key={listing.id} md={5} sm={7} xs={10}>
             <ListingItem listing={listing} />
           </Grid>
         ))}
+      {data && data.listings.length == 0 && (
+        <>
+          <Grid item>
+            <Typography variant="body1" align="center">
+              Det er for øyeblikket ingen verv tilgjengelige.
+            </Typography>
+          </Grid>
+          <Grid item md={6} xs={10}>
+            <Image src={_EmptyStreet} alt="Empty street" />
```

Comment:
Should be blank for decorative images
```suggestion
            <Image src={_EmptyStreet} alt="" />
```